### PR TITLE
feat: [CDS-35460]: added cloudformation pipeline icons

### PR DESCRIPTION
**Pull request by willd0yle**
You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

File: src/icons/cloud-formation-create-light.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...-rule="evenodd" clip-rule="evenodd" d="M4.291 16.086c0-6.605 5.355-11.96 11.96-11.96s11.96 5.355 11.96 11.96a.52.52 0 1 0 1.04 0c0-7.18-5.82-13-13-13s-13 5.82-13 13 5.82 13 13 13a.52.52 0 1 0 0-1.04c-6.605 0-11.96-5.354-11.96-11.96Zm6.504-.485a.52.52 0 0 1 .52-.52h3.93v-3.93a.52.52 0 1 1 1.04 0v3.93h3.93a.52.52 0 1 1 0 1.04h-3.93v3.93a.52.52 0 0 1-1.04 0v-3.93h-3.93a.52.52 0 0 1-.52-.52Z" fill="#fff"/><path d="m21.891 27.948.094.107 4.756 1.348.021-.836-3.33-.836-.163-.058-1.378.275Z" fill="#fff"/><path d="m31.54 27.948-4.857 1.62-.014-1 3.33-.837.145-.163 1.396.38Z" fill="#FAFCF9"/><path d="m26.669 22.221 3.33.776 1.465-.07.076-.146-4.64-1.67-.231 1.11Z" fill="#C3C4C3"/><path d="m23.432 27.731-1.541.217-.043-.16V22.95l.043-.169 1.392.104.149.112v4.734Z" fill="#FBFCF9"/><path d="m26.762 22.221-3.33.776-1.54-.216.057-.091 4.791-1.364.022.895Z" fill="#C3C4C3"/><path d="m26.709 18.623-5.531 2.792.002 7.9 5.529 2.79.082-.124V18.785l-.082-.162ZM25.09 28.908l-3.2-.96V22.78l3.2-.96v7.087ZM30 27.731l1.54.217.087-.153v-4.867l-.087-.147-1.54.217v4.733Z" fill="#DBDCD9"/><path d="m26.709 32.106 5.542-2.79.002-7.9-5.544-2.793v13.483Zm1.63-10.285 3.2.96v5.167l-3.2.96v-7.087Z" fill="#fff"/></svg>
```
Comment (abhinavrastogi-harness) [inline src/icons/cloud-formation-create-light.svg:1]: can we use one svg with css controlled colors for light and dark variations?

**Review by abhinavrastogi-harness — APPROVED** [review]

File: src/icons/cloud-formation-create-light.svg (hunk 1 above)
Comment (willd0yle) [reply src/icons/cloud-formation-create-light.svg:1]: We could, I added both sets as it they were both given to me by UX/UI. I have no preference to using css to control a light version. 

I will remove the light version. We can style with css when needed.

**Comment by willd0yle** [comment]
retrigger ImageSnapshot

**Comment by willd0yle** [comment]
retrigger ImageSnapshot

**Review by abhinavrastogi-harness — APPROVED** [review]

**Pull request merged**